phoenix-core/src/it/java/org/apache/phoenix/end2end/AlterTableIT.java

```diff
@@ -1795,4 +1796,65 @@ public void testChangePropertiesUpdatesLASTDDLTimestamp() throws Exception {
                     newLastDDLTimestamp > oldLastDDLTimestamp);
         }
     }
-}
\ No newline at end of file
+
```

Comment:
What about if someone sets MAX_LOOKBACK_AGE to NULL?

Reply:
Currently, for most table level properties setting them to NULL via Alter is no-op. Created a JIRA issue: https://issues.apache.org/jira/browse/PHOENIX-7161 for the same. Thanks